fdmi/plugins/sched.c

```diff
@@ -334,86 +325,80 @@ static void usage(void)
  * @retval 0      Success.
  * @retval -Exxx  Error.
  */
-
-static int
-sched_args_parse(struct sched_conf *params, int argc, char ** argv)
+static int sched_args_parse(struct sched_conf *params, int argc, char ** argv)
 {
-	int    rc = 0;
+	int rc = 0;
 
 	params->local_addr 	= NULL;
 	params->ha_addr    	= NULL;
 	params->profile_fid     = NULL;
 	params->process_fid   	= NULL;
+	params->fdmi_plugin_fid_s = NULL;
 
 	rc = M0_GETOPTS("m0sched", argc, argv,
 			M0_HELPARG('?'),
 			M0_VOIDARG('i', "more verbose help",
-					LAMBDA(void, (void) {
-						usage();
-						exit(0);
-					})),
+				   LAMBDA(void, (void) {
+						   usage();
+						   exit(0);
+					   })),
 			M0_STRINGARG('l', "Local endpoint address",
-					LAMBDA(void, (const char *string) {
-					params->local_addr = (char*)string;
-					})),
+				     LAMBDA(void, (const char *string) {
+						     params->local_addr = (char*)string;
+					     })),
 			M0_STRINGARG('h', "HA address",
-					LAMBDA(void, (const char *str) {
-						params->ha_addr = (char*)str;
-					})),
+				     LAMBDA(void, (const char *str) {
+						     params->ha_addr = (char*)str;
+					     })),
 			M0_STRINGARG('f', "Process FID",
-					LAMBDA(void, (const char *str) {
-						params->process_fid = (char*)str;
-					})),
-			M0_STRINGARG('p', "Profile options for Client",
-					LAMBDA(void, (const char *str) {
-						params->profile_fid = (char*)str;
-					})),
+				     LAMBDA(void, (const char *str) {
+						     params->process_fid = (char*)str;
+					     })),
+			M0_STRINGARG('p', "Profile options for client",
+				     LAMBDA(void, (const char *str) {
+						     params->profile_fid = (char*)str;
+					     })),
 			M0_STRINGARG('g', "FDMI plugin fid",
-					LAMBDA(void, (const char *str) {
-						params->fdmi_plugin_fid_s =
-							(char*)str;
-					})));
+				     LAMBDA(void, (const char *str) {
+						     params->fdmi_plugin_fid_s =
+							     (char*)str;
+					     })));
 	if (rc != 0)
 		return M0_ERR(rc);
-	rc = m0_fid_sscanf(params->fdmi_plugin_fid_s, &params->fdmi_plugin_fid);
-	if (rc != 0) {
-		return M0_ERR_INFO(rc, "invalid format: fdmi_plugin_fid=%s",
-		                   params->fdmi_plugin_fid_s);
-	}
-	/* All mandatory params must be defined. */
+
+        /* All mandatory params must be defined. */
 	if (rc == 0 &&
 	    (params->local_addr == NULL || params->ha_addr == NULL ||
 	     params->profile_fid == NULL || params->process_fid == NULL ||
 	     params->fdmi_plugin_fid_s == NULL)) {
 		usage();
-		rc = M0_ERR(-EINVAL);
+		return M0_ERR(-EINVAL);
+	}
+
+	rc = m0_fid_sscanf(params->fdmi_plugin_fid_s, &params->fdmi_plugin_fid);
+	if (rc != 0) {
+		rc = M0_ERR_INFO(rc, "Invalid fdmi plugin fid format: fdmi_plugin_fid=%s",
+				 params->fdmi_plugin_fid_s);
 	}
 
 	return rc;
 }
 
 static int classify_handle_fdmi_rec_not(struct m0_uint128 *rec_id,
```

Comment:
how about name it `process_fdmi_record() ` ?

Reply:
ok